napalm/base/base.py

```diff
@@ -178,12 +178,16 @@ def compare_config(self):
         """
         raise NotImplementedError
 
-    def commit_config(self):
+    def commit_config(self, confirmed=None, message=None):
```

Comment:
We should document what "confirmed" is supposed to be.

Reply:
I also really think we shouldn't name this variable `confirmed`. It is a terrible name; `revert_in` or `timer` are way better names.

Reply:
`confirmed` is logical and is the most widely adopted name for this function. Basically saying you have to `confirm` this commit in how many minutes.

I think we will probably just add more confusion by naming it something else.

But agreed...it needs documented.

Reply:
+1 on `revert_in`, describes the intent better. `confirmed` nomenclature maybe better known by network engineers, but I wouldn't cater to them exclusively with this library, you may have engineers without that background.